tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/util/ServerUtil.java

```diff
@@ -654,7 +655,7 @@ public boolean accept( File dir, String name )
 
                         String name = entry.getName();
 
-                        if( name.contains( ".web" ) )
+                        if( name.endsWith( ".jar" ) )
```

Comment:
I changed here as we found there are some bundles have jsp files without "web" in the name, like com.liferay.journay.term.of.use